WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -112,6 +113,9 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main_activity);
 
+        // Allow vector drawables on pre-Lollipop (i.e. < API 21).
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
```

Comment:
Note I added the same line in: https://github.com/wordpress-mobile/WordPress-Android/pull/4950

From the [doc](https://developer.android.com/reference/android/support/v7/app/AppCompatDelegate.html#setCompatVectorFromResourcesEnabled(boolean)):
> Please note: this only takes effect in Activities created after this call.

I think it's better to have this call in `WordPress.onCreate`, in case we use selectors in one of the tab fragment at some point or in the login screen.


From the [same doc](https://developer.android.com/reference/android/support/v7/app/AppCompatDelegate.html#setCompatVectorFromResourcesEnabled(boolean)), it's also a bit worrying:
> This feature defaults to disabled, since enabling it can cause issues with memory usage, and problems updating Configuration instances. If you update the configuration manually, then you probably do not want to enable this. You have been warned.